src/olympia/activity/templates/activity/emails/from_reviewer.txt

```diff
@@ -14,6 +14,6 @@ To respond, please reply to this email or visit {{ url }}. {% if is_info_request
 
 Thank you for your attention.
 
--- 
+--
```

Comment:
😕 I'm pretty sure I didn't make a random whitespace change here.  Might be git and/or my editor stripping the trailing space.  @wagnerand can you remember if this trailing space is required?  I seem to remember some odd specification about sig separators.

Reply:
That doesn't ring a bell for me.

Reply:
See https://www.ietf.org/rfc/rfc3676.txt section 4.3, I don't know how required it is these days but at least the spec says that it is.